dev-embedded/ponyprog/files/ponyprog-3.0.0-install_doc.patch

```diff
@@ -0,0 +1,36 @@
+--- ponyprog-3.0.0/distribution/posix/CMakeLists.txt-	2018-01-18 21:12:21.504884644 +0300
++++ ponyprog-3.0.0/distribution/posix/CMakeLists.txt	2018-01-18 22:02:34.575994692 +0300
+@@ -33,25 +33,25 @@
+         ) 
+ 
+ INSTALL(DIRECTORY "${PROJECT_SOURCE_DIR}/distribution/innosetup/" 
+-        DESTINATION "share/doc/ponyprog/" 
++        DESTINATION "share/doc/ponyprog-3.0.0/" 
```

Comment:
As @dracwyrm suggested, please use the `GNUInstallDirs` module, use `${CMAKE_INSTALL_DOCDIR}` here and upstream the resulting patch